awx/ui_next/src/util/useDebounce.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import useDebounce from './useDebounce';
+
+function TestInner() {
+  return <div />;
+}
+function Test({ fn, delay = 500, data }) {
+  const debounce = useDebounce(fn, delay);
+  debounce(data);
+  return <TestInner />;
+}
+
+test('useDebounce', () => {
+  jest.useFakeTimers();
+  const fn = jest.fn();
+  expect(fn).toHaveBeenCalledTimes(0);
```

Comment:
you should put this first assertion after `mount`

Reply:
@keithjgrant updated.